### PR TITLE
Environment-setup, switched to functional

### PR DESCRIPTION
**Pull request by Renzlawr**
Had some blockers, figured it all out and got everything working good.

### CODE REVIEW

File: app/javascript/components/App.js [hunk 1]
```diff
@@ -10,29 +10,25 @@ import NotFound from './pages/NotFound'
 import {
   BrowserRouter as Router,
   Route,
-  Switch
+  Routes
```
Comment (SunkissedQueen) [inline app/javascript/components/App.js:13]: Is Routes for the newer version of react-router-dom?

File: spec/requests/apartments_spec.rb [hunk 2]
```diff
@@ -0,0 +1,28 @@
+require 'rails_helper'
+
+RSpec.describe "Apartments", type: :request do
+  describe "GET /index" do
+    it 'returns a list of apartments' do
+    
+      user = User.where(email: 'test@example.com').first_or_create(password: '12345678', password_confirmation: '12345678')
+
+      user.apartments.create(
```
Comment (SunkissedQueen) [inline spec/requests/apartments_spec.rb:9]: Pass some values on the object instead of the boilerplate info

**Review by SunkissedQueen — CHANGES_REQUESTED** [review]
Please pass values on the object you are creating for the test. Ping me when you have made that change.

Good work on the switching to functions. Here's a link to other functional component syntax. https://www.delftstack.com/howto/react/export-default-in-react/#:~:text=Default%20Exports%20in%20React,function%2C%20like%20the%20example%20below.

File: app/javascript/components/App.js (hunk 1 above)
Comment (Renzlawr) [reply app/javascript/components/App.js:13]: Yeah Routes replaces Switch in react-router-doms latest version

File: spec/requests/apartments_spec.rb [hunk 3]
```diff
@@ -4,19 +4,19 @@
...
-        bedrooms: integer, 
-        bathrooms: integer, 
-        pets: string,
-        image: text
+        street: "2942 ratopolis lane",
+        city: "Los Angeles",
+        state: "California",
+        manager: "Rat Lord",
+        email: "biggestratwhoeatsallofthecheese@gmail.com", 
+        price: "10 blocks of swiss",
```
Comment (SunkissedQueen) [inline spec/requests/apartments_spec.rb:15]: 😆

**Review by SunkissedQueen — APPROVED** [review]

**Pull request merged**